08_arrays/main.go

```diff
@@ -43,45 +43,230 @@ func main() {
+	// print result
+	var intArray []int
+	intArray = append(intArray, 1, 2, 3, 4, 5, 6, 7, 8, 9, 9)
+	// debug
+	fmt.Println("Print intArray:", intArray)
+
+	// create new empty array of 10 strings
+	var lengthIntArray []string
+	// counter
+	var arrayAssemblyCounter = 0
+
+	// loop to create output array
+	for {
+		// if counter is > than the last/[10] array element break the loop
+		if arrayAssemblyCounter > intArray[arrayAssemblyCounter]-1 {
+			break
+		}
+		// append "x" to the array, by calling the function with a loop that returns the concat'd string
+		lengthIntArray = append(lengthIntArray, concatVariable(intArray[arrayAssemblyCounter]))
+		// debug
+		fmt.Println("intArray element:", intArray[arrayAssemblyCounter])
+		// increase counter to run with the next array element
+		arrayAssemblyCounter++
+	}
+	// print result array, task req, DONE
+	fmt.Println(lengthIntArray)
+
 	// TODO: Create 2 arrays with 10 integers.
 	// Create 3 more arrays:
 	// - One with the sum of the elements of the 2 arrays.
 	// - One with the difference of the elements of the 2 arrays.
 	// - One with the multiplication of the elements of the 2 arrays.
 	// E.g. [1, 2, 3] and [4, 5, 6] => [5, 7, 9], [-3, -3, -3], [4, 10, 18]
-	// Print the 3 new arrays.
+	//
+	//	// Print the 3 new arrays.
+
+	// create int array with 10 elements
+	var anotherIntArray []int
+	// fill it
+	anotherIntArray = append(anotherIntArray, 1, 4, 5, 4, 6, 8, 5, 13, 5, 12)
+	// create another
+	var yetAnotherIntArray []int
+	// fill it
+	yetAnotherIntArray = append(yetAnotherIntArray, 23, 4, 1, 1, 3, 5, 6, 4, 6, 2)
+
+	// create array for sums of elements of the 2 arrays
+	var sumArray []int
+	// create array for difference of elements of the 2 arrays
+	var diffArray []int
+	// // create array for multiplication of elements of the 2 arrays
+	var multArray []int
+
+	// create loop for element sum calculation
+	// counter
+	var sumCounter int = 0
+	for {
+		// TODO: replace hard coded value with last index of
+		if sumCounter > 9 {
+			break
+		}
+		// append sum of element of arrays
+		sumArray = append(sumArray, anotherIntArray[sumCounter]+yetAnotherIntArray[sumCounter])
+		// increase counter
+		sumCounter++
+	}
+	// print result sumArray
+	fmt.Println(sumArray)
+
+	// create loop for element difference calculation
+	// counter
+	var diffCounter int = 0
+	for {
+		// TODO: replace hard coded value with last index of
+		if diffCounter > 9 {
+			break
+		}
+		diffArray = append(diffArray, anotherIntArray[diffCounter]-yetAnotherIntArray[diffCounter])
+		// increase counter
+		diffCounter++
+	}
+	// print result diffArray
+	fmt.Println(diffArray)
+
+	// create loop for element multiplication calculation
+	// counter
+	var multCounter = 0
+	for {
+		// TODO: replace hard coded value with last index of
+		if multCounter > 9 {
+			break
+		}
+		multArray = append(multArray, anotherIntArray[multCounter]*yetAnotherIntArray[multCounter])
+		// increase counter
+		multCounter++
+	}
```

Comment:
make this with just one loop, the extra 2 are redundant

Reply:
Done, good one. Felt a bit dirty writing it, thought of variables, but this is better.